### PR TITLE
PP-896: pbs_snapshot Option -d <pbs_diag> is not working

### PR DESCRIPTION
**Pull request by agrawalravi90**
#### Issue-ID

* **[PP-896](https://pbspro.atlassian.net/browse/PP-896)**

#### Problem description
* pbs_snapshot Option -d <pbs_diag> is not working

#### Cause / Analysis
* The -d option was not implemented in the main RFE. After some discussion, it was decided that it shouldn't be added now as pbs_diag is being deprecated anyways. If there's a need to convert diags to snapshots, we should just write a script.

#### Solution description
* Removed vestigial code that referred to the -d option from the codebase.

#### Testing
* Since no new functionality was added/removed and only dead code was removed, I just tested the change using pbs_snapshot's test suite.

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [X] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by agrawalravi90** [comment]
Thanks Arun. Requesting one more review, it's a tiny PR, thanks!

File: test/fw/ptl/utils/pbs_snaputils.py [hunk 1]
```diff
@@ -501,8 +497,8 @@ def __init__(self, out_dir, diag_path=None, server_host=None,
         self.mapfile = map_file
 
         # Initialize Server and Scheduler objects
-        self.server = Server(server_host, diag=self.diag)
-        self.scheduler = Scheduler(server=self.server, diag=self.diag)
+        self.server = Server(server_host)
+        self.scheduler = Scheduler(server=self.server)
```
Comment (bhroam) [inline test/fw/ptl/utils/pbs_snaputils.py]: You're no longer calling this with self.diag, but I didn't see self.diag get removed.  Is it still being set or initialized somewhere?

**Comment by bhroam** [comment]
Ahh, there it is :)  I'm fine with it.  I sign off.

**Comment by agrawalravi90** [comment]
Thanks Bhroam! I'll rebase

**Comment by agrawalravi90** [comment]
@PBSPro/pbspro-maintainers this PR is ready for merge, thanks!

**Pull request merged**